frontend/src/modules/user/views/userProfileView.js

```diff
@@ -69,10 +66,17 @@ define(function(require){
 
     togglePasswordView: function() {
       event && event.preventDefault();
+      
+      var passwordField = document.querySelector('.reveal-input-password');
```

Comment:
In adapt this type of selector isn't common, we tend to use jQuery selectors, please could you update this

```
var passwordField = this.$('.reveal-input-password');
var passwordReveal = this.$('.reveal-password-button');
```

You'll have to use attr like so

`passwordField.attr('type', 'text');`